packages/pwa-kit-react-sdk/src/ssr/server/react-rendering.js

```diff
@@ -260,7 +260,7 @@ const renderApp = (args) => {
             React.cloneElement(el, {
                 ...el.props,
                 ...scriptProps,
-                src: el.props.src && getAssetUrl(el.props.src.slice(1))
+                src: el.props.src
```

Comment:
```suggestion
```
We don't need this line because it's already done in the previous line on 261 `...el.props`